docs/setup-systems.md

```diff
@@ -144,3 +145,8 @@ In your `Dockerfile` copy `clever-tools` from the image itself with a simple one
 ```Dockerfile
 COPY --from=clevercloud/clever-tools /bin/clever /usr/local/bin/clever
 ```
+
+### Nix package manager
+
+If you are using Nix or NixOS, the package is available on [nixpkgs - unstable](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=clever-tools).
+You may add it to your configuration just like any other package.
```

Comment:
```suggestion
[Add it](LINK) to your configuration.
```
If there is no specific command for that, replace LINK with a link to Nix doc about it.

Reply:
LGTM, my main concern here is to redirect to Nix documentation for new users to know where to find more informations.

Reply:
Found a better solution, the nixpkgs search result shows a "how to install section". It wasn't available at the time of the PR but now it is so I'll link to that:
![nixpkgs search result for clever-tools - how to install section](https://github.com/CleverCloud/clever-tools/assets/100240294/357bca5c-e0e0-440a-8e56-947de76df19c)

https://search.nixos.org/packages?channel=unstable&show=clever-tools&from=0&size=50&sort=relevance&type=packages&query=clever-tools